client/src/app/components/Toolbar/Toolbar.jsx

```diff
@@ -24,21 +24,23 @@ import Overlay, { OverlayStyleType } from 'common/components/Overlay';
 class Toolbar extends PureComponent {
   state = {
     cohortFormVisibility: false,
+    formDescription: '',
+    formTitle: '',
     overlayVisibility: false,
     shoppingFormVisibility: false,
     userBarMenuVisibility: false
   };
 
-  componentDidMount() {
-    document.addEventListener('click', this.clickListener);
-    document.addEventListener('keydown', this.escapeListener);
-  }
-
   componentWillUnmount() {
     document.removeEventListener('click', this.clickListener);
     document.removeEventListener('keydown', this.escapeListener);
   }
```

Comment:
these listeners won't be removed because creationForms are always "mounted" in the toolbar (they are only not visible)